cmd/server/version.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	BuildID = "nil"
+)
+
+func versionCmd() *cobra.Command {
```

Comment:
I think we want BuildID to be available more generally, and not just in the command line.
It could be *used* by the version command. 

For example when we add a `/healtz`    or `/info` endpoint it needs to report the same info.
I suggest we move the `BuildID` to   `pkg/aspect. BuildID` or similar.